### PR TITLE
First Release

### LINKED ISSUE #17: PyPI release
Heya there

I know this project is pretty young, but is there a chance of a pypi release, in line with the readme?

```
$ pip3 install crispy_tailwind

ERROR: Could not find a version that satisfies the requirement crispy_tailwind (from versions: none)
ERROR: No matching distribution found for crispy_tailwind
``` 

I know it's possible to `-e` the git URL in requirements.txt, but in a Docker build that results in the dependency being installed at the user, not system level, making it effectively invisible to the app.

Cheers!

### PR DESCRIPTION
**Pull request by smithdc1**
@carltongibson 

Appreciate your thoughts on this. We have tests that show an opinionated rendered for the crispy filters and for a few of the basic layout objects. I think more will work but need to work on tests / docs. 

Is it worth issuing now, seems to be getting a bit of interest and I've not made the readme as clear as I could have done.

A couple of questions:

- Is this 0.1... or should we use calver?
- I've tagged as pre-alpha... but 🤷 

Closes #17

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# CHANGELOG for crispy-tailwind
+
+## 0.1 (2020-06-09)
+
```
Comment (carltongibson) [inline CHANGELOG.md:4]: I’d say something like “First Release, please do come and test” in here.

**Review by carltongibson — APPROVED** [review]
I think with the Pre-Alpha tag in place it’s fine. Folks are obviously keen. Put it out there. 🙂

Super effort. 🥳

**Comment by smithdc1** [comment]
Thank you. 

All ready to go, fingers crossed I'll get time to publish this evening.

**Comment by smithdc1** [comment]
It's up! https://pypi.org/project/crispy-tailwind/

🥳

**Pull request merged**